templates/localgov_alert_banner.html.twig

```diff
@@ -9,14 +9,47 @@
  * Available variables:
  * - content: A list of content items. Use 'content' to print all content, or
  * - attributes: HTML attributes for the container element.
+ * - display_title: Boolean indicating if alert banner title should be displayed
+ * - remove_hide_link: Boolean indicating if the link to close the banner should be hidden
+ * - type_of_alert: Type of alert banner
  *
  * @see template_preprocess_localgov_alert_banner()
  *
  * @ingroup themeable
  */
 #}
-<div{{ attributes.addClass('localgov_alert_banner') }}>
-  {% if content %}
-    {{- content -}}
-  {% endif %}
+{{ attach_library('localgov_alert_banner/alert_banner') }}
+{% set has_link = content.link is not empty %}
+{% set type_of_alert =  type_of_alert|split('--')[1] %}
+{% set classes = [
+    'wrapper',
+    'js-alert-banner',
+    'localgov_alert_banner--' ~ type_of_alert,
+    is_front ? 'localgov_alert_banner--homepage' : '',
+    has_link ? 'localgov_alert_banner--has-link' : 'localgov_alert_banner--no-link'
+  ]
+%}
+<!-- Alert Banner-->
+<div class="localgov_alert_banner" role="alert">
+  <div{{ attributes.addClass(classes) }}>
+    <div class="localgov_alert_banner--inner">
+      <div class="localgov_alert_banner--inner--content">
+        {% if display_title %}
+          <h2 class="localgov_alert_banner--content-heading">{{ content.title }}</h2>
+        {% endif %}
+        <p class="localgov_alert_banner--content-body">{{ content.short_description }}</p>
```

Comment:
As its now a wysiwyg element, should be a `<div>`.
Also as users might add new fields, I'd use {{ content|without(link) }}

Reply:
We might also consider just rending `{{ content }}` here instead of the link separately now that we are controlling the link title in a preprocesss (the special theming for the link is a very BHCC thing).

Reply:
This is done, I've left the link as rendered separately for now, to make styling more straightforward in case the short_description includes links.